scalafix-core/shared/src/main/scala/scalafix/internal/config/NoInferConfig.scala

```diff
@@ -0,0 +1,22 @@
+package scalafix.internal.config
+
+import metaconfig._
+import org.langmeta._
+
+case class NoInferConfig(
+    extraSymbols: List[Symbol] = Nil
+) {
+  implicit val reader: ConfDecoder[NoInferConfig] =
+    ConfDecoder.instanceF[NoInferConfig] { c =>
+        c.getOrElse("extraSymbols")(extraSymbols).map(xs => NoInferConfig(xs))
+    }
+  implicit val symbolReader: ConfDecoder[Symbol] =
```

Comment:
There is a Symbol.Global reader in `scalafix.internal.config` which we should use instead, it handles a few corner cases.